docs/tasks/debug-application-cluster/events-stackdriver.md

```diff
@@ -0,0 +1,84 @@
+---
+assignees:
+- crassirostris
+- piosz
+title: Events Using Stackdriver
+---
+
+Kubernetes events are objects that provide an insight into what is happening
+inside a cluster, for example, what decisions were made by scheduler or why
+some pods were evicted from the node. You can read more about using events
+for debugging your application in the [Application Introspection and Debugging
+](docs/tasks/debug-application-cluster/debug-application-introspection)
+section.
+
+Since events are API objects, they are stored in the apiserver on master. To
+avoid filling up master's disk, a retention policy is enforced: events are
+removed after one hour after the last occurence. To provide longer history
+and aggregation capabilities, a third party solution should be installed,
```

Comment:
The idea was "install 3rd party solution and export events there for future use", not "install 3rd party solution, capture events there and then exported them from there". Changed to `a third party solution should be installed to capture events`